### DISCUSSION TITLE
Support for new GitHub projects

### DISCUSSION THREAD

**Discussion by milogert** [pwntester/octo.nvim#414] [Q&A]
Is there any plans to add support for the new GitHub projects? At the moment they don't seem to be detected in the PR or Issue views (i.e. they just don't show up) and adding something as a card to a project only lets you browse the classic projects.

**Comment by pwntester** [comment]
Sorry but I dont have plans to add support for them as I dont have much time. PRs are welcome and I can try to assist as much as I can
